### PR TITLE
minor readability changes to best practice names

### PR DESCRIPTION
**Pull request by js-kyle**
Very minor changes, such as removing unneeded capitalization

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,11 +150,11 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/><br/>
 
-## ![✔] 2.6 Shut the process gracefully when a stranger comes to town
+## ![✔] 2.6 Shutdown the process gracefully when a stranger comes to town
```
Comment (BrunoScheufler) [inline README.md]: Not sure, maybe `Shut down` instead of `Shutdown`?

Comment (js-kyle) [reply README.md]: Hmm how about swapping it for 'exit'?

**Comment by js-kyle** [comment]
Cheers @BrunoScheufler, I've made the update we discussed

File: README.md (hunk 1 above)
Comment (i0natan) [reply README.md]: +1 for shut down 

Using bold and interesting words contributes to text attractiveness, It's not only about using the right grammar and vocabulary

**Comment by i0natan** [comment]
@js-kyle @BrunoScheufler Most of the improvements LGTM, I love the perfectionism here... :]

**Comment by i0natan** [comment]
@BrunoScheufler @js-kyle Shouldn't we always assign to someone?

**Comment by js-kyle** [comment]
@BrunoScheufler Your call on Shut down vs exit - I can either change that, or merge this?

**Comment by i0natan** [comment]
🎆

**Pull request merged**